android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java

```diff
@@ -196,7 +194,7 @@ else if(this.options.fileCache)
             if (this.options.trusty) {
                 clientBuilder = RNFetchBlobUtils.getUnsafeOkHttpClient();
```

Comment:
This line should have been `clientBuilder = RNFetchBlobUtils.getUnsafeOkHttpClient(client);` so it can build on top of the `react-native` `client`. I must have forgotten to change it. @wkh237 could you change it on master or should I submit a new PR?